portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -195,12 +195,49 @@ definition {
 	}
 
 	@description = "Verify that inputs to the Body Text can be undone and redone"
-	@ignore = "true"
 	@priority = "5"
 	test CanUndoAndRedo {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131245 CanUndoAndRedo pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
```

Comment:
For line 210:

Please refer to https://github.com/liferay-forms/liferay-portal/pull/233#discussion_r632481834 

Should change `fieldPositionNumber` to 0.